docs/blog/2017-09-26-embracing-graphql/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Embracing GraphQL - How I migrated to Gatsby v1"
+date: "2017-09-26"
+image: "toa-heftiba-274947.jpg"
+author: "Tim Arney"
+excerpt: "Moving to Gatsby 1.x and embracing the new GraphQL setup can be daunting but the effort is going to payoff"
+---
+
+Before I built React FAQ, I went looking for a static site generator.
+
+I had a few criteria in mind.
+
+* Fast (hopefully a given for a static site)
+* Easy to use
+* React based
+* Easy to update
+
+Gatsby at the time was still pre-1.0 but it's workflow met my needs. I loved that I could drop my existing Markdown files into a pages directory and Gatsby would do the rest.
+
+Webpack loader —> React.js wrapper component —> static HTML page. Done.
+
+Done until I read this [Github Issue](https://github.com/gatsbyjs/gatsby/issues/420) ... New GraphQL data layer.
+
+![issue-420](issue-420.png) 
+
+Things in Gatsby were about to change. I read on to see how easy it’d be to upgrade.
+
+Having never used GraphQL, the new “routeQuery” in components looked very foreign.
+
+```javascript
+export const routeQuery = `
+{
+  allMarkdown {
+    edges {
+      node {
+        path
+        frontmatter {
+          title
+        }
+      }
+    }
+  }
+}
+```
+
+What happened to my “drop files into a folder and call it a day” setup?  What is all this weird syntax @kylemathews.
+
+Reading the issue Kyle's reasoning made sense but this didn't seem to be simple at all. 0.x seemed more “automatic” to me.  Drop in your files and go.
+
+
+## 1.0 Released
+
+When 1.0 landed, I waited for a while before trying to update my website.  The reviews we're solid but I was still stuck on my pre 1.0 mindset.
+
+Then on Sep 8 2017 I saw this tweet from @ryanflorence
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">.<a href="https://twitter.com/gatsbyjs">@gatsbyjs</a> is a completely original approach to web development. The mix of static/dynamic and how it handles data is really interesting.</p>&mdash; Ryan Florence (@ryanflorence) <a href="https://twitter.com/ryanflorence/status/906233888560758784">September 8, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
This can be removed as we load the twitter script elsewhere.

Reply:
Just remove the script tag and keep the blockquote (embed)?

Reply:
Yup!